pkgs/development/python-modules/sqlalchemy-utils/default.nix

```diff
@@ -0,0 +1,40 @@
+{ lib, isPy3k, fetchPypi, buildPythonPackage
+, six, sqlalchemy
+, pytz, mock, pytest, flexmock, python-dateutil }:
+
+buildPythonPackage rec {
+  pname = "sqlalchemy-utils";
+  version = "0.33.6";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "SQLAlchemy-Utils";
+    sha256 = "45ab41c90bfb8dd676e83179be3088b3f2d64b613e3b590187163dd941c22d4c";
+  };
+
+  propagatedBuildInputs = [
+    six
+    sqlalchemy
+  ];
+
+  # TypeError: sort_query() missing 1 required positional argument: 'query'
+  doCheck = !isPy3k;
```

Comment:
If I build `python2.pkgs.sqlalchemy-utils`, I get `Ran 0 tests in 0.000s`. A custom `checkPhase` might be needed, probably `py.test sqlalchemy_utils tests`

Reply:
Well quite a lot of tests pass ... while a lot of tests also fail ...